### PR TITLE
Update release note format

### PR DESCRIPTION
**Pull request by LitoMore**
I updated the release note format to similar to [`npm/cli` release note](https://github.com/npm/cli/releases/tag/v10.4.0).

The benefit is the `Contributors` section in the release note.

I also removed the `core.debug(file)`. It's too long and super hard to read. We can simply use the GitHub REST API to check specific PR response. Or use our `npm run preview` script for debugging.

### CODE REVIEW

File: test/create.test.js
Comment (adamrusted) [inline test/create.test.js:1]: Is it worth having more than one maintainer named here - just to validate?

Comment (LitoMore) [reply test/create.test.js:1]: GitHub REST API does not support co-author field.

In the meantime, I prefer to close a stale PR and create new one instead.

However, You can still update the release note in its created release PR.

Comment (adamrusted) [reply test/create.test.js:1]: I meant having more than one author in our test script, to prove that it doesn't mix up PR creators.

**Review by mondeja — APPROVED** [review]
Looks good to me though I like the @adamrusted's suggestion.

File: __mocks__/@actions/github.js [hunk 1]
```diff
@@ -42,132 +42,151 @@ const PRs = [
...
     base: { ref: 'master' },
+    user: { login: 'LitoMore' },
   },
   {
     // Previous release
     title: 'Release',
     number: 499,
     merged_at: '2011-01-02T19:01:12Z',
     base: { ref: 'master' },
+    user: { login: 'LitoMore' },
```
Comment (adamrusted) [inline __mocks__/@actions/github.js:189]: Added a variety of maintainer names to the mock data.
To be applied along with the update to tests, also listed.

```suggestion
    number: 500,
    merged_at: null,
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    //  1: PR that adds a file
    title: '501',
    number: 501,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    //  2: PR that modifies an SVG
    title: '502',
    number: 502,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    //  3: PR that was merged before the previous release
    title: '498',
    number: 498,
    merged_at: '2011-01-01T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    //  4: PR that modifies one icon's color
    title: '503',
    number: 503,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    //  5: PR that modifies one icon's source
    title: '504',
    number: 504,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'fbernhart' },
  },
  {
    //  6: PR that modifies one icon's color and source
    title: '505',
    number: 505,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    //  7: PR that modifies an SVG and modifies that icon's color
    title: '506',
    number: 506,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    //  8: PR that modifies an SVG and modifies that icon's source
    title: '507',
    number: 507,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    //  9: PR that modifies an SVG and modifies that icon's color and source
    title: '508',
    number: 508,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    // 10: PR that adds an SVG and modifies another icon's color and source
    title: '509',
    number: 509,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    // 11: PR that adds an SVG and modifies another SVG
    title: '510',
    number: 510,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'fbernhart' },
  },
  {
    // 12: PR that changes the data file but no metadata
    title: '511',
    number: 511,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    // 13: PR that adds a brand with a lowercased name
    title: '512',
    number: 512,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    // 14: PR that adds ACM (which should be listed before the lowercased brand)
    title: '513',
    number: 513,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    // 15: PR that adds a brand with an accented name
    title: '514',
    number: 514,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    // 16: PR that removes an icon
    title: '515',
    number: 515,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    // Skipped release
    title: '[skip] Skip this one',
    number: 516,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'master' },
    user: { login: 'fbernhart' },
  },
  {
    // Previous release
    title: 'Release',
    number: 499,
    merged_at: '2011-01-02T19:01:12Z',
    base: { ref: 'master' },
    user: { login: 'LitoMore' },
```

File: test/create.test.js [hunk 2]
```diff
@@ -10,28 +10,28 @@ The new version will be: **v2.0.0**
...
-- Foo (#501)
-- Jest (#509)
-- Pokémon (#514)
-- WordPress (#510)
+- ACM (#513) (@LitoMore)
+- bar (#512) (@LitoMore)
+- Foo (#501) (@LitoMore)
+- Jest (#509) (@LitoMore)
+- Pokémon (#514) (@LitoMore)
+- WordPress (#510) (@LitoMore)
```
Comment (adamrusted) [inline test/create.test.js:18]:
```suggestion
- ACM (#513) (@mondeja)
- bar (#512) (@LitoMore)
- Foo (#501) (@LitoMore)
- Jest (#509) (@service-paradis)
- Pokémon (#514) (@PeterShaggyNoble)
- WordPress (#510) (@fbernhart)
```

File: test/create.test.js [hunk 3]
```diff
@@ -10,28 +10,28 @@ The new version will be: **v2.0.0**
...
-- Postman (#506)
+- 1Password (#503) (@LitoMore)
+- Abstract (#509) (@LitoMore)
+- AddThis (#508) (@LitoMore)
+- Adobe (#504) (@LitoMore)
+- Feedly (#502) (@LitoMore)
+- Intel (#507) (@LitoMore)
+- Mozilla (#505) (@LitoMore)
+- Opera (#510) (@LitoMore)
+- Postman (#506) (@LitoMore)
```
Comment (adamrusted) [inline test/create.test.js:30]:
```suggestion
- 1Password (#503) (@service-paradis)
- Abstract (#509) (@service-paradis)
- AddThis (#508) (@PeterShaggyNoble)
- Adobe (#504) (@fbernhart)
- Feedly (#502) (@mondeja)
- Intel (#507) (@mondeja)
- Mozilla (#505) (@adamrusted)
- Opera (#510) (@fbernhart)
- Postman (#506) (@LitoMore)
```

File: test/create.test.js [hunk 4]
```diff
@@ -10,28 +10,28 @@ The new version will be: **v2.0.0**
...
 
 # Removed Icons
 
-- Foobar (#515)
+- Foobar (#515) (@LitoMore)
```
Comment (adamrusted) [inline test/create.test.js:34]:
```suggestion
- Foobar (#515) (@service-paradis)
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Varying the maintainer names used, to properly demonstrate functionality works.

**Pull request merged**